add Okta SSO info (#626)
neo4j/docs-aura: modules/ROOT/pages/security/single-sign-on.adoc

This will result in Azure asking for consent to display details related to these scopes. 
For more information, see link:https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes#standard-claims[OpenID Connect Scopes]

== Okta SSO 

. In the *Okta admin portal* go to *Applications* and then *Create App Integration*

. For *Sign-in method* select *OIDC - OpenID Connect*. 

. For *Application type* select *Web Application*.

. Select *Next*.

. For *Grant type* select *Authorization Code*.

. For *Sign-in redirect URIs* add https://login.neo4j.com/login/callback as the redirect URI

. Save 

. In the Aura console, go to *Organization Settings* > *Security* > *Single Sign On* > *New configuration*

. Select if you want the SSO configuration to be applied to organization-level logins (which acts as a login to the Aura console), to specific projects within the organization (where login is on the instance), or both.

. For IdP Type select *Okta*

. For Client ID enter the Okta *Client ID* 

. For Client Secret enter the *Client Secret*

> . For Discovery URI take the domain from your Okta portal which should be something like https://dev-123-admin.okta.com/ and add `.well-known/openid-configuration` so the final URL should look something like https://dev-123-admin.okta.com/.well-known/openid-configuration

Reviewer: Clarify that you either enter discovery URI, or
- Issuer
- Authorization endpoint
- etc.

not both.

You could add a sentence: You also have option to press "Manual Configuration" and enter Issuer, ... or something like that.

Reviewer: Consider doing the same for Entra

This will result in Azure asking for consent to display details related to these scopes. 
For more information, see link:https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes#standard-claims[OpenID Connect Scopes]

== Okta SSO 

> . In the *Okta admin portal* go to *Applications* and then *Create App Integration*

Reviewer 2: Overall, we need to be consistent with the punctuation. So I'd say, use full stop everywhere.

. For *Sign-in method* select *OIDC - OpenID Connect*. 

. For *Application type* select *Web Application*.

. Select *Next*.

. For *Grant type* select *Authorization Code*.

. For *Sign-in redirect URIs* add https://login.neo4j.com/login/callback as the redirect URI

. Save 

. In the Aura console, go to *Organization Settings* > *Security* > *Single Sign On* > *New configuration*

. Select how you want the SSO configuration to be applied in Aura:

.. *Use as a log in method for the organization* applies to organization-level logins (which acts as a login to the Aura console).

.. *Use as a login method for instances within Projects in this Org* applies to the project-level and you can select specific projects within the organization (where login is on the instance).

.. Or, you can select both

. For IdP Type select *Okta*

. For Client ID enter the Okta *Client ID* 

. For Client Secret enter the *Client Secret*

. Choose discovery method

> .. For Discovery URI take the domain from your Okta portal which should be something like https://dev-123-admin.okta.com/ and add `.well-known/openid-configuration` so the final URL should look something like https://dev-123-admin.okta.com/.well-known/openid-configuration

Reviewer 2: ```suggestion
.. For Discovery URI take the domain from your Okta portal which should be something like https://dev-123-admin.okta.com/ and add `.well-known/openid-configuration`.
The final URL should look similar to `https://dev-123-admin.okta.com/.well-known/openid-configuration`.
```

.. Alternatively, you can select *Manual Configuration* and enter the values separately, including Issuer, Authorization Endpoint, Token Endpoint and JWKS URI.

. Configure any additional settings as needed

> ..  For organization-level SSO, no additional settings needed

Reviewer 2: ```suggestion
..  For organization-level SSO, no additional settings needed.
```

> .. For project-level SSO, enter role mappings if applicable

Reviewer 2: ```suggestion
.. For project-level SSO, enter role mappings if applicable.
```
